src/parallel/job_token.rs

```diff
@@ -238,9 +238,10 @@ mod inprocess_jobserver {
             // Note that we could use `num_cpus` here but it's an extra
             // dependency that will almost never be used, so
             // it's generally not too worth it.
-            let mut parallelism = 4;
-            // TODO: Use std::thread::available_parallelism as an upper bound
-            // when MSRV is bumped.
+
+            let mut parallelism = std::thread::available_parallelism()
+                .map(|v| usize::from(v) as u32)
+                .unwrap_or(4);
             if let Ok(amt) = var("NUM_JOBS") {
```

Comment:
Thanks! Could I get you to:
1. Not invoke `available_parallelism` if `NUM_JOBS` is set.
2. Update the comments above (they're now outdated, as we wouldn't want to use `num_cpus`).

Reply:
Sure, will do! I tried to write the whole function as one long functional chain, but it wasn't super pretty :D I'll try to find some middle ground.